index.js

```diff
@@ -110,7 +110,19 @@ function dragDrop (elem, listeners) {
     }
 
     // file drop support
-    if (e.dataTransfer.items) {
+    if (e.dataTransfer.files) {
+      var files = e.dataTransfer.files
+
+      if (files.length === 0) return
+
+      for (var i = 0; i < files.length; ++i) {
+        files[i].fullPath = '/' + files[i].name
```

Comment:
It wasn't necessary to change the code style here.

Reply:
It was, a `FileList` isn't an array, it doesn't have a `forEach` method.